fusillade/api/users/__init__.py

```diff
@@ -1,21 +1,29 @@
 from flask import request, make_response, jsonify
 from fusillade import User, directory
+from fusillade.utils.authorize import assert_authorized
 
 
-def put_new_user():
+def put_new_user(token_info:dict):
     json_body = request.json
+    assert_authorized(token_info['https://auth.data.humancellatlas.org/email'],
+                      ['fus:PutUser'],
+                      [f'arn:hca:fus:*:*:user/{json_body["user_id"]}/'])
```

Comment:
`arn:hca:fus:*:*:user/` seems more appropriate, since the user/{user_id} resource doesn't exist yet.

Reply:
also is the trailing `/` needed?

Reply:
updated